libkbfs/tlf_handle_resolve.go

```diff
@@ -688,8 +687,8 @@ func (ra resolvableAssertion) resolve(ctx context.Context) (
 			id:   id,
 		}, keybase1.SocialAssertion{}, tlfID, nil
 	case NoSuchUserError:
-		socialAssertion, ok := externals.NormalizeSocialAssertion(ra.assertion)
-		if !ok {
+		socialAssertion, serr := ra.resolver.NormalizeSocialAssertion(ctx, ra.assertion)
```

Comment:
I think we should just call the error `err` and return it below.

Reply:
i left it as the original implementation since there are some error handlers that depend on this error type and i didn't want to subtly change the functionality. should we just log `serr` and return `err`?

Reply:
Hmm, ok, that makes sense.  Unfortunately we don't have easy access to a proper logger here.  Just add a comment about why we're dropping it?

Reply:
will do!